tree: delete scripts that were moved to cosa (#156)
coreos/fedora-coreos-releng-automation: coreos-ostree-importer/README.md

## Fedora Messaging sender from the Fedora CoreOS Pipeline

> In the cosa repo, you will find a script called `fedmsg-fcos-pipeline-ostree-import-request`. It is used by the

Reviewer: ```suggestion
In the COSA repo, you will find a script called [`fedmsg-send-ostree-import-request`](https://github.com/coreos/coreos-assembler/blob/main/src/fedmsg-send-ostree-import-request). It is used by the
```

[Fedora CoreOS Pipeline](https://github.com/coreos/fedora-coreos-pipeline.git)
to send the request to the importer.

> Here's how you might send a request using `fedmsg-fcos-pipeline-ostree-import-request`:

Reviewer: ```suggestion
Here's how you might send a request using `fedmsg-send-ostree-import-request`:
```

> ```

Reviewer: Can we update the below example:

- new name of file
- use `buildfetch` rather than `buildprep`

## Fedora Messaging sender from the Fedora CoreOS Pipeline

In the coreos-assembler repo, you will find a script called
[`fedmsg-send-ostree-import-request`](https://github.com/coreos/coreos-assembler/blob/main/src/fedmsg-send-ostree-import-request).
It is used by the [Fedora CoreOS Pipeline](https://github.com/coreos/fedora-coreos-pipeline.git) to send the request to the importer.

Here's how you might send a request using `fedmsg-send-ostree-import-request`:

```
cosa buildfetch --build=31.20200212.20.0

> /usr/lib/coreos-assembler/send-ostree-import-request \

Reviewer: ```suggestion
/usr/lib/coreos-assembler/fedmsg-send-ostree-import-request \
```
